modules/opts.nix

```diff
@@ -47,6 +63,15 @@ in {
     )
     optionsAttrs;
 
+  imports =
+    map
+    ({
+      from,
+      to,
+    }:
+      mkRenamedOptionModule [from] [to])
+    legacyAliases;
+
```

Comment:
Maybe I'm nitpicking too much, but I would rather not have the code for deprecation warnings splitted.

We could group it at the beginning with something like:
```nix
imports = mapAttrsToList
  (old: new:
    mkRenamedOptionModule [old] [new]
  )
  {
    options = "opts";
    globalOptions = "globalOpts";
    localOptions = "localOpts";
  };
```

Reply:
> Maybe I'm nitpicking too much

Not at all. Happy to fit in with your preferred code style.

Done.